packages/ui-components/src/components/ui/card.tsx

```diff
@@ -9,7 +9,7 @@ const Card = React.forwardRef<
   <div
     ref={ref}
     className={cn(
-      'rounded-lg border bg-card text-card-foreground shadow-sm',
+      'rounded-lg border border-border bg-card text-card-foreground shadow-sm',
```

Comment:
Nit: is border name of a color we have? Otherwise can prob delete this class

Reply:
yeah it's a shadcn token